packages/jellyfish-transaction-builder/src/txn/txn_builder_liq_pool.ts

```diff
@@ -50,4 +50,18 @@ export class TxnBuilderLiqPool extends P2WPKHTxnBuilder {
       changeScript
     )
   }
+
+  /**
+   * Create pool pair.
+   *
+   * @param {PoolCreatePair} create pool pair
+   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
+   * @returns {Promise<TransactionSegWit>}
+   */
+  async create (create: PoolCreatePair, changeScript: Script): Promise<TransactionSegWit> {
```

Comment:
Looks like we did not have `create` in liq_pool tx builder. should add a test file similar to `packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_liq_pool_remove_liquidity.test.ts`

Reply:
yea agree.. but that one gonna be settled on another PR exclusively